host/4/publish.yml

```diff
@@ -234,9 +234,38 @@ steps:
       docker push $TARGET_REGISTRY/powershell:$(TargetVersion)-powershell7-appservice-quickstart
 
       docker system prune -a -f
-    displayName: tag and push powershell images
+    displayName: tag and push powershell 70 images
```

Comment:
should this be 72 images

Reply:
The section above is to build the PowerShell 7.0 image, so I added `70` to the display name.